File: Iniciante/PHP/exercise_1050.php
<?php

function getCidade($ddd) {
    $cidade = "DDD nao cadastrado";
    switch ($ddd) {
        case 61:
            $cidade = "Brasilia";
            break;
        case 71:
            $cidade = "Salvador";
            break;
        case 11:
            $cidade = "Sao Paulo";
            break;
        case 21:
            $cidade = "Rio de Janeiro";
            break;
        case 32:
            $cidade = "Juiz de Fora";
            break;
        case 19:
            $cidade = "Campinas";
            break;
        case 27:
            $cidade = "Vitoria";
            break;
        case 31:
            $cidade = "Belo Horizonte";
            break;
    }

    return $cidade;
}

$ddd = intval(fgets(STDIN));

$cidade = getCidade($ddd);

print_r($cidade."\n");

File: Iniciante/Python/exercises from 1000 to 1099/exercise_1050.py
dddInformado = int(input())
ddds = {
    '61': 'Brasilia',
    '71': 'Salvador',
    '11': 'Sao Paulo',
    '21': 'Rio de Janeiro',
    '32': 'Juiz de Fora',
    '19': 'Campinas',
    '27': 'Vitoria',
    '31': 'Belo Horizonte'
}

if str(dddInformado) in ddds:
    print(ddds[str(dddInformado)])
else:
    print('DDD nao cadastrado')